integration_test/third_party_apps_test/applications/couchdb/centos_rhel/install

```diff
@@ -40,10 +47,34 @@ cat << EOF > local.ini
 [vhosts]
 [admins]
 admin = otelp
+[log]
+writer = file
+file = /var/log/couchdb/couchdb.log
 EOF
 sudo mv local.ini /opt/couchdb/etc/local.ini
 sudo chown couchdb:couchdb /opt/couchdb/etc/local.ini
 sudo chmod 0755 /opt/couchdb/etc/local.ini
 
 sudo systemctl enable couchdb 
 sudo systemctl restart couchdb
+
+timeout 60s bash <<EOF
+wait_for_couchdb() {
+    until curl http://admin:otelp@localhost:5984 > /dev/null 2>&1
+    do
+        echo "Waiting for couchdb to start. . ."
+        sleep "1"
+    done
+}
+
+wait_for_couchdb
+
+sleep 5
+
+echo "couchdb started"
+EOF
+
+# https://docs.couchdb.org/en/stable/setup/single-node.html
+curl -X PUT http://admin:otelp@localhost:5984/_users
+curl -X PUT http://admin:otelp@localhost:5984/_replicator
+curl -X PUT http://admin:otelp@localhost:5984/_global_changes
```

Comment:
[Optional] We have many instances of such replicated code that is not OS-specific. I wonder if it's time to add the ability to specify a common post-install script, and run it after the OS-specific install one… Probably beyond the scope of this PR, though.

Reply:
Yeah, a refactor would be valuable here, but not in scope for this PR IMO.

Reply:
Opened b/370089941 to track.